### PR TITLE
chore: fix spelling errors

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: .github/workflows/_build.yaml [hunk 1]
```diff
@@ -91,7 +91,7 @@ jobs:
         GITHUB_TOKEN: ${{ github.token }}
 
     # Generate the requirements.txt that contains the hash digests of the dependencies and
-    # generate the SBOM using CyclonDX SBOM generator.
+    # generate the SBOM using CycloneDX SBOM generator.
```
Comment (jsoref) [inline .github/workflows/_build.yaml:94]: brand

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -97,7 +97,7 @@ jobs:
       contents: read
       packages: read
 
-  # Create a new Release on Github from the verified build artifacts, and optionally
+  # Create a new Release on GitHub from the verified build artifacts, and optionally
```
Comment (jsoref) [inline .github/workflows/release.yaml:100]: brand

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
 - resolve podman compatibility issues (#512)
 - do not use git set-branches if the target branch is not currently available in the repository (#491)
-- fix bash syntax error when running `run_macaron.sh` on MacOS (#528)
+- fix bash syntax error when running `run_macaron.sh` on macOS (#528)
```
Comment (jsoref) [inline CHANGELOG.md:24]: brand (note that this spelling was never correct, the names were `Mac OS` and later `Mac OS X`)

File: THIRD_PARTY_LICENSES.txt [hunk 4]
```diff
@@ -5324,7 +5324,7 @@ Copyright 2014 The Go Authors.  All rights reserved.
 Copyright 2014 The Go Authors. All rights reserved.
 Copyright 2014 The Kubernetes Authors.
 Copyright 2014 The go-github AUTHORS. All rights reserved.
-Copyright 2014 Unknwon
+Copyright 2014 Unknown
```
Comment (jsoref) [inline THIRD_PARTY_LICENSES.txt:5327]: Lawyers tend to get annoyed at changes to license files...

That said, unless there's an entity named `Unknwon`, this is problematic

File: THIRD_PARTY_LICENSES.txt [hunk 5]
```diff
@@ -5434,7 +5434,7 @@ Copyright 2020 Google LLC.
 Copyright 2020 Google LLC. All Rights Reserved.
 Copyright 2020 Google LLC. All Rights Reserved.\n" +
 Copyright 2020 Gregor Martynus
-Copyright 2020 Intel Coporation.
+Copyright 2020 Intel Corporation.
```
Comment (jsoref) [inline THIRD_PARTY_LICENSES.txt:5437]: I'm confident that this name was wrong. Although I'm not sure where it's originally from. https://www.intel.com/content/www/us/en/legal/trademarks.html

File: docs/source/pages/using.rst [hunk 6]
```diff
@@ -102,7 +102,7 @@ To simplify the examples, we use the same configurations as above if needed (e.g
 
   pkg:<git_service_hostname>/<organization>/<name>
 
-The list bellow shows examples for the corresponding PURL strings for different git repositories:
+The list below shows examples for the corresponding PURL strings for different git repositories:
```
Comment (jsoref) [inline docs/source/pages/using.rst:105]: `bellow` is technically a word, it just isn't right in programming contexts...

File: docs/source/pages/using.rst [hunk 7]
```diff
@@ -332,7 +332,7 @@ Macaron's policy engine accepts policies specified in `Datalog <https://en.wikip
 can verify if a project and all its dependencies pass certain checks. We use `Soufflé <https://souffle-lang.github.io/index.html>`_
 as the Datalog engine in Macaron. Once you run the checks on a target project as described :ref:`here <analyze-command>`,
 the check results will be stored in ``macaron.db`` in the output directory. We pass the check results to the policy engine by providing the path to ``macaron.db`` together with a Datalog policy file to be validated by the policy engine.
-In the Datalog policy file, we must specify the identifier for the target software component that we are interested in to validate the policy against. These are two ways to specify the target software component in the Datalog policy file:
+In the Datalog policy file, we must specify the identifier for the target software component that interests us to validate the policy against. These are two ways to specify the target software component in the Datalog policy file:
```
Comment (jsoref) [inline docs/source/pages/using.rst:335]: this whole sentence is still hard to parse, but I think this is easier than the original...

File: src/macaron/slsa_analyzer/provenance/intoto/v1/__init__.py [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
-"""This module handles in-toto version version 1 attestations."""
+"""This module handles in-toto version 1 attestations."""
```
Comment (jsoref) [inline src/macaron/slsa_analyzer/provenance/intoto/v1/__init__.py:4]: ?

File: tests/conftest.py [hunk 9]
```diff
@@ -344,7 +344,7 @@ def circle_ci_service(setup_test):  # type: ignore # pylint: disable=unused-argu
 
 @pytest.fixture()
 def gitlab_ci_service(setup_test):  # type: ignore # pylint: disable=unused-argument
-    """Create a GitlabCI service instance.
+    """Create a GitLabCI service instance.
```
Comment (jsoref) [inline tests/conftest.py:347]: brand...

File: tests/dependency_analyzer/cyclonedx/test_cyclonedx.py [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
-"""This module tests the CyclondeDX helper functions."""
+"""This module tests the CycloneDX helper functions."""
```
Comment (jsoref) [inline tests/dependency_analyzer/cyclonedx/test_cyclonedx.py:4]: Differrent misspelling of the same brand...

File: tests/slsa_analyzer/checks/test_build_service_check.py [hunk 11]
```diff
@@ -179,7 +179,7 @@ def test_build_service_check(self) -> None:
         # Use pip as a module in CI with invalid goal to build the artifact.
         no_pip_interpreter_build_ci = MockAnalyzeContext(macaron_path=MacaronTestCase.macaron_path, output_dir="")
         no_pip_interpreter_build_ci.dynamic_data["build_spec"]["tools"] = [pip]
-        bash_commands["commands"] = [["python", "-m", "pip", "installl"]]
+        bash_commands["commands"] = [["python", "-m", "pip", "install-"]]
```
Comment (jsoref) [inline tests/slsa_analyzer/checks/test_build_service_check.py:182]: This should have the same result while not itself including a word not in the dictionary.

File: tests/slsa_analyzer/checks/test_infer_artifact_pipeline.py [hunk 12]
```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
-"""This module contains tests for the Infer ArtiFact Pipeline check."""
+"""This module contains tests for the Infer Artifact Pipeline check."""
```
Comment (jsoref) [inline tests/slsa_analyzer/checks/test_infer_artifact_pipeline.py:4]: ?

File: tests/slsa_analyzer/git_service/test_github.py [hunk 13]
```diff
@@ -24,6 +24,6 @@ def test_is_detected(self) -> None:
         assert github.is_detected("git@github.com:7999/org/name")
         assert github.is_detected("ssh://git@github.com:7999/org/name")
         assert not github.is_detected("http://gitlab.com/org/name")
-        assert not github.is_detected("git@githubb.com:org/name")
-        assert not github.is_detected("git@not_supported_git_host.com:7999/org/name")
+        assert not github.is_detected("git@github0.com:org/name")
+        assert not github.is_detected("git@not-supported-git-host.com:7999/org/name")
```
Comment (jsoref) [inline tests/slsa_analyzer/git_service/test_github.py:28]: `_` isn't valid in .com dns names...

too many typos mask other problems....

fwiw, I have an additional commit not included in this PR that would add `github.example.com` to this block, as it's a more interesting check.

File: tests/slsa_analyzer/provenance/expectations/cue/test_cue_validator.py [hunk 14]
```diff
@@ -12,16 +12,17 @@
 from macaron.database.table_definitions import CUEExpectation
 from macaron.slsa_analyzer.provenance.expectations.cue.cue_validator import get_target, validate_expectation
 
-EXPEC_RESOURCE_PATH = Path(__file__).parent.joinpath("resources")
+EXPECT_RESOURCE_PATH = Path(__file__).parent.joinpath("resources")
 PROV_RESOURCE_PATH = Path(__file__).parent.parent.parent.joinpath("resources")
+PACKAGE_URLLIB3 = "pkg:github.com/urllib3/urllib3"
```
Comment (jsoref) [inline tests/slsa_analyzer/provenance/expectations/cue/test_cue_validator.py:17]: some linter objected to the natural change which apparently caused the line to exceed a limit. This avoids that.

**Comment by jsoref** [comment]
image

File: THIRD_PARTY_LICENSES.txt (hunk 4 above)
Comment (jsoref) [reply THIRD_PARTY_LICENSES.txt]: Apparently this is https://github.com/Unknwon (presumably Grafana or similar). It's unfortunate that it isn't an email address or name as opposed to an amorphous thing, but whatever. Dropping.

File: THIRD_PARTY_LICENSES.txt (hunk 5 above)
Comment (jsoref) [reply THIRD_PARTY_LICENSES.txt:5437]: This appears to come from kubernetes, I've submitted a PR to fix it there.

**Comment by behnazh-w** [comment]
@jsoref Thanks a lot for the PR. This is awesome. Can you please approve the Contributor Agreement?

**Comment by jsoref** [comment]
Or

**Comment by jsoref** [comment]
I was able to sign it using a desktop browser, but please get someone to fix the flow so that it doesn't require one.

**Comment by behnazh-w** [comment]
@jsoref Sorry to hear that. I have escalated the issue to the team that is in charge of the OCA app.

**Comment by jsoref** [comment]
Fwiw, I actually hit a bunch of other errors including things that looked like capacity issues or general outages.

There's no particular indication of what to actually expect going forward. At this time, my thing just says "Under review".

File: THIRD_PARTY_LICENSES.txt (hunk 5 above)
Comment (jsoref) [reply THIRD_PARTY_LICENSES.txt:5437]: And it was merged

File: src/macaron/slsa_analyzer/provenance/intoto/v1/__init__.py (hunk 8 above)
Comment (behnazh-w) [reply src/macaron/slsa_analyzer/provenance/intoto/v1/__init__.py:4]: Yes, that's correct.

**Review by behnazh-w — APPROVED** [review]

**Comment by behnazh-w** [comment]
@jsoref Looks like your [commits](https://github.com/oracle/macaron/pull/612/commits) are not signed. Can you please [sign them](https://github.com/oracle/macaron/blob/staging/CONTRIBUTING.md#contributing-code) and also include the `Signed-off-by:` line?

File: src/macaron/output_reporter/templates/base_template.html [hunk 15]
```diff
@@ -256,7 +256,7 @@
         }
 
         /*
-            The reason why we need to create a separate .toggler class is because all .caret class are set binded
+            The reason why we need to create a separate .toggler class is because all .caret class are bound
```
Comment (benmss) [inline src/macaron/output_reporter/templates/base_template.html:259]:
```suggestion
            The reason we need to create a separate .toggler class is that the .caret class is bound
```

**Comment by jsoref** [comment]
Do you mean GPG signed? That's a bridge too far (the number of hoops and stumbling blocks for this project is absurd). I have an expired signature from 2018 and this is the first project to demand that since it expired. 

I have an ssh signing key added in 2022 that I'd be willing to use.

Plus, you merged #613 w/o requiring that.

None of these changes should even require concern about (c) as they're all trivial.

**Comment by behnazh-w** [comment]
We have turned on a check on GitHub to verify commit signatures. As long as that check passes, it's good. Note that the check has passed for your other PR: 
image

But the commits on this PR are missing the `verified` green tick for some reason.

**Comment by jsoref** [comment]
I think the check just needs to be rerun. 

It also should be improved so that you can easily determine what it's actually complaining about.

**Comment by jsoref** [comment]
Oh, your docs are wrong. That commit was signed w/ an ssh signature:
image

Sure, I can do that here. But you should fix the docs to clarify what it actually tolerates (either "whatever github tolerates" or "gpg / ssh" if it's specifically picky)

**Comment by behnazh-w** [comment]
> Sure, I can do that here. But you should fix the docs to clarify what it actually tolerates (either "whatever github tolerates" or "gpg / ssh" if it's specifically picky)

Thanks for the suggestion. I have created an [issue](https://github.com/oracle/macaron/issues/616) for it.

**Comment by behnazh-w** [comment]
@jsoref The commits are now signed and the GitHub check passes, thanks. There is only one [comment](https://github.com/oracle/macaron/pull/612#discussion_r1464121004) left by @benmss that needs to be addressed.

**Review by benmss — APPROVED** [review]

**Review by behnazh-w — APPROVED** [review]

**Comment by spavlusieva** [comment]
hi @jsoref, We apology for the inconvenience you've experienced while signing an OCA, and are working on fixing these issues.
